doc/manual/manual.tex

```diff
@@ -8058,6 +8058,8 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 \textit{This section is a co-production of Cedric Thieulot, Juliane Dannberg,
 Timo Heister and Wolfgang Bangerth with an extension to this benchmark provided by the Virginia Tech Department of Geosciences class ``Geodynamics and ASPECT'' co-taught by Scott King and D.~Sarah Stamps.}
 
+\paragraph{Computation of the van Keken Problem with compositional fields}
+\label{paragraph:van-keken compositional fields}
```

Comment:
Just drop the `\paragraph` and use the label from the subsubsection instead (see also my comment below about the new paragraph you add).